pkgs/development/tools/clpm/default.nix

```diff
@@ -25,8 +25,8 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     runHook preBuild
 
-    ln -s ${openssl.out}/lib/libcrypto.so.* .
-    ln -s ${openssl.out}/lib/libssl.so.* .
+    ln -s ${openssl.out}/lib/libcrypto.* .
```

Comment:
```suggestion
    ln -s ${openssl.out}/lib/libcrypto.${stdenv.hostPlatform.extensions.sharedLibrary}.* .
```

Reply:
Is targetPlatform not better here?

The naming convention is unfortunately not the same between Linux and Darwin:
 - `libssl.so.1.1`
 - `libssl.1.1.dylib`

So I guess we'd have to do:

```sh
ln -s ${openssl.out}/lib/libcrypto*${stdenv.targetPlatform.extensions.sharedLibrary}* .
```

Reply:
The convention is build/host/target. The target only applies to building cross compilers.

I think, `*.${ext}*` is still better than just `*`. It, at the very least, signals the intention to only link dynamic libraries.

Reply:
Ah, gotcha. Looks like it has to be `*${ext}*`, though, because it includes the dot (`.so`, `.dylib`). 😕

Would be nice to have a Linux review on this to verify.